Packs/Intezer/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,14 @@
+### Integrations
+
+#### Intezer v2
+
+- New Integration Param - `AnalyzeBaseURL` - Allowes setting the integration to work with a different Intezer Analyze instance (i.e on premise)
+- Modifying the integration to XSOAR integration development best practices
+- Fixes for the context not joining together
+- Integration now uses the Intezer Analyze Python SDK for future support and updates
+
+- New Commands - 
+    - `intezer-get-sub-analyses` - Get a list of the Analysis sub analyses
+    - `intezer-get-analysis-code-reuse` - Get the Code reuse information for an analyses
+    - `intezer-get-analysis-metadata` - Get an Analysis metdata (different file hashes and more)
+    - `intezer-get-family-info` - Get family information from Intezer Analyze
```

Comment:
Add here your new outputs.

Reply:
Do you have an example? non of the last change logs contained "outputs"